src/WebJobs.Script.NuGet/WebJobs.Script.nuspec

```diff
@@ -16,12 +16,12 @@
     <dependencies>
       <dependency id="Microsoft.Azure.WebJobs" version="2.0.0-alpha-10331" />
```

Comment:
Hmm, if that is the case I wonder why VS did not pull in the correct dependency when upgrading the extensions.